message_definitions/v1.0/uAvionix.xml

```diff
@@ -118,5 +130,17 @@
       <description>Transceiver heartbeat with health report (updated every 10s)</description>
       <field type="uint8_t" name="rfHealth" enum="UAVIONIX_ADSB_RF_HEALTH" display="bitmask">ADS-B transponder messages</field>
     </message>
+    <message id="10004" name="UAVIONIX_ADSB_OUT_CFG_REGISTRATION">
+      <description>Aircraft Registration.</description>
+      <field type="char[9]" name="registration">Aircraft Registration (ASCII string A-Z, 0-9 only), e.g. "N8644B ". Trailing spaces (0x20) only. This is null-terminated.</field>
```

Comment:
We're using "callsign" in `UAVIONIX_ADSB_OUT_CFG` for this, I think?  We should be consistent in our naming of the same data.  And our descriptions for that matter, given the trailing spaces comment here.  This also affects this message's name.

```suggestion
      <field type="char[9]" name="registration">Aircraft Registration (ASCII string A-Z, 0-9 only), e.g. "N8644B ". Spaces (0x20) can only appear at end of string. This is null-terminated.</field>
```

Similarly for the other 2 messages here.

Reply:
I pulled that text from their datasheet